src/driver.rs

```diff
@@ -268,14 +268,14 @@ pub fn main() {
 
             // Setting RUSTC_WRAPPER causes Cargo to pass 'rustc' as the first argument.
             // We're invoking the compiler programmatically, so we ignore this/
-            let wrapper_mode = Path::new(&orig_args[1]).file_stem() == Some("rustc".as_ref());
+            let wrapper_mode = orig_args.get(1).map(Path::new).and_then(Path::file_stem) == Some("rustc".as_ref());
```

Comment:
I think you should rebase on the current master.

Reply:
Sure.

Reply:
Rebased, but seems that current master [is broken](https://github.com/rust-lang/rust-clippy/runs/310926218) as well.